src/labels/CustomLabels.labels

```diff
@@ -1771,6 +1779,14 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>alloObjectUnsupported</shortDescription>
         <value>The Manage Allocations button is only supported from a Campaign, Opportunity or Recurring Donation record.</value>
     </labels>
+    <labels>
+        <fullName>alloPaymentNotEnabled</fullName>
+        <categories>Allocation, Error</categories>
+        <language>en_US</language>
+        <protected>false</protected>
```

Comment:
@pedrodalcol Can you update any new custom labels to be marked as Protected=True? This is part of our standard to ensure translations work as expected.

Reply:
And so that we can delete the label in the future if necessary!